exercises/Daniel_Murphy/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,151 @@
+#include <iostream>
+#include <fstream>
+#include <map>
+#include <string>
+#include <vector>
+#include <algorithm>
+#include <functional>
+using namespace std;
+
+class ReadFile
```

Comment:
I'm not sure the class adds a lot of value here. You may as well just have a couple of standalone functions I think.